packages/@sanity/default-login/src/LoginWrapper.js

```diff
@@ -24,11 +24,11 @@ export default class LoginWrapper extends React.PureComponent {
     ]).isRequired,
     title: PropTypes.node,
     description: PropTypes.node,
-    sanityLogo: PropTypes.oneOfType([
+    SanityLogo: PropTypes.oneOfType([
       PropTypes.node,
       PropTypes.func
     ]),
```

Comment:
I think the backward-compatibility is missing here?

Reply:
Also, maybe make a console.warn when sanityLogo is provided mentioning that the prop i deprecated?